scalableminds/webknossos
# Send Email when new Organization is created

### URL of deployed dev instance (used for testing):
- https://neworganizationmail.webknossos.xyz

### Steps to test:
- set up wK with Mailer
- disable insertion of initial data, empty your database
- change the receiver adress in application.conf
- in onboarding flow, create new organization
- check your emails

### Issues:
- fixes #2931

------
